frameworks/Java/ninja-standalone/src/main/resources/META-INF/persistence.xml

```diff
@@ -20,15 +20,16 @@
                 <property name="hibernate.jdbc.batch_size" value="100" />
 
                 <!-- Connection Pooling settings -->
-                <property name="hibernate.connection.provider_class"
-                          value="org.hibernate.service.jdbc.connections.internal.C3P0ConnectionProvider" />
-
-                <property name="hibernate.c3p0.max_size" value="100" />
-                <property name="hibernate.c3p0.min_size" value="0" />
-                <property name="hibernate.c3p0.acquire_increment" value="1" />
-                <property name="hibernate.c3p0.idle_test_period" value="300" />
-                <property name="hibernate.c3p0.max_statements" value="0" />
-                <property name="hibernate.c3p0.timeout" value="100" />     
```

Comment:
Did you run mvn clean install? This should run tests and tests should pass without any issues... But it did not work for me... I had to rewrite the h2 part to:

        <persistence-unit name="h2" transaction-type="RESOURCE_LOCAL">
            <provider>org.hibernate.ejb.HibernatePersistence</provider>
            <properties>
                <property name="javax.persistence.provider" value="org.hibernate.ejb.HibernatePersistence" />
                <property name="hibernate.connection.driver_class" value="org.h2.Driver" />
                <property name="hibernate.dialect" value="org.hibernate.dialect.H2Dialect" />
                <!-- you may want to enable the ddl if you do not use migrations. -->
                <property name="hibernate.hbm2ddl.auto" value="create" />
                <property name="hibernate.show_sql" value="false" />
                <property name="hibernate.format_sql" value="false" />
                <property name="hibernate.jdbc.batch_size" value="100" />

                <!-- Connection Pooling settings -->
                <property name="hibernate.connection.provider_class" value="org.hibernate.hikaricp.internal.HikariCPConnectionProvider" />
            </properties>
        </persistence-unit>